docs/development-guide.md

```diff
@@ -38,7 +30,7 @@ Copy the contents and use that as password / secret key. And run that again to g
 
 ## Input variables
 
-The generator (Cookiecutter) will ask you for data on a long list of fields which will be used to populate variables across the project, customising it for you out the box. **You might want to have these on hand before generating the project.**
+The generator (Cookiecutter) will ask you for data on a long list of fields which will be used to populate variables across the project, customising it for you out the box. **You might want to have these on hand before generating the project.** Take note, some of these inputs are production deployment specific, so you can opt to fill them later or rewrite them in the provided `cookiecutter-config-file.yml` and [supplying it as a parameter in a new cookiecutter call](../{{cookiecutter.project_slug}}/README.md). 
```

Comment:
"customising" -> "customizing" for consistency with US English.

Reply:
How would a user fill in the production-specific ones later?

Reply:
"fill them" -> "fill them in"

Reply:
> How would a user fill in the production-specific ones later?

I went ahead and added that they can replace them in the `.env` file.